src/components/NcActionTextEditable/NcActionTextEditable.vue

```diff
@@ -258,10 +258,10 @@ $input-margin: 4px;
 
 		// bottom-right corner
 		position: absolute;
-		right: $icon-margin + 1;
+		right: calc($icon-margin + 1);
```

Comment:
we should get rid of this options in the future (`...-right`, `...-left`, `right`, `left`) to properly support RTL layout and just use `...-inline-start` `...-inline-end`.

Just a note for the future, not related to this PR :sweat_smile: